frontends/web/src/components/backups/create.jsx

```diff
@@ -74,6 +75,10 @@ export default class Create extends Component {
                 alert(data.errorMessage); // eslint-disable-line no-alert
             } else {
                 this.props.onCreate();
+                console.log(data);
```

Comment:
Is this log still necessary? And doesn't ESLint complain?

Reply:
No, good catch.

CI passed, so I guess not ;)